### DISCUSSION TITLE
Looking for camera recommendation

### DISCUSSION THREAD

**Discussion by Ysbrand** [blakeblackshear/frigate#4960] [Ask A Question]
I've been using Frigate for a year or so, and unfortunately one of my camera's is dying. So I need to replace it.

I'm looking for a POE camera (outside) and I'm currently debating between the following 2:
Amcrest IP8M-2496EW-V2
Reolink RLC-811A

But maybe I'm overlooking something so i fyou ahve an alternative in more or less the same price range, I would appreciate that as well.

Thanks

**Comment by thinkloop** [comment]
I personally wouldn't recommend Reolink, it's the only camera out of many that causes me problems with smudging, smearing and artifacts, and it randomly does H265 for one resolution but changes to H264 for another without any indication. They try to do more but don't seem equipped to achieve it. I've enjoyed Lorex cameras. At the same time I have no experience with the specific models you outlined.

**Comment by drocanyd** [comment]
I have 17 5MP bullet IP cameras model VC-EB05SG-L running flawlessy for months on my frigate install here.   They are unbranded version of the Dahua camera I purchased off ebay.

**Comment by dz4news** [comment]
I have been using the Dahua cameras listed under the recommendation page of the documentation. They are a little pricey, but the night vision is beyond comparison to anything I have seen. I started with Reolink and sold all of them... there were a pain to configure.

**Comment by gururise** [comment]
Stay away from Reolink.  Purchased 3 of their RLC-520 cams about a year ago, and within one year, all three of them are dead. Two of them died after a rainstorm, and the third one died randomly.

**Comment by leccelecce** [comment]
I'm mainly using Hikvision cameras (the DS-2CD2xxxG2-yy line) which have been straightforward to set up, have a good range of configurable options, good multiple stream support and have been rock solid. 
Slightly lower end than the Dahua 5442 from my research but otherwise good. You do have to spend time understanding the model numbers though - the G2's are the first in that range that have an acceptable sensor size to megapixel ratio.

Biggest issue I have with them for my use case (identification) is that due to the factory setup and wide apertures the close focus range isn't great, resulting in slightly soft images sometimes. Not everyone is willing to take a camera apart to adjust the focus!

Disclaimer: relatively new to this, and Hik cameras are widely available used here in the UK
